docs/gameplay-features/spawners.md

```diff
@@ -0,0 +1,24 @@
+---
+description: Information on obtaining and using spawners in Theatria.
+---
+
+# Spawners
+
+Spawners are a valuable resource in Theatria, allowing players to generate specific mobs. Here's how you can obtain and use them effectively.
+
+## How to Obtain Spawners
+
+1. **Ranks**
+   Unlock spawners through [ranks](ranks/ethereal-ranks), which can be purchased with in-game currency. Higher ranks often include better rewards.
+
+## Placement Restrictions
+
+- **Mining World Only**: Spawners can only be placed in the [Mining World](../worlds-dimensions/mining). 
+- **World Reset**: The Mining World typically resets once a month, near the beginning of the month. Check the #announcements channel in our Discord server for updates.
+
+## Tips
+
+- Save spawners for strategic use when you plan to spend time in the Mining World.
+- When you place a spawner in the Mining World, you can break it with a pickaxe. Using a pickaxe enchanted with Silk Touch increases the chance of keeping and reusing the spawner.
```

Comment:
Can we mention the chance of losing the spawner, there is a 75% chance of success when picking up a spawner.